src/file.rs

```diff
@@ -57,3 +108,55 @@ pub fn write_file(path: impl AsRef<Path>, overwrite: bool) -> Result<File> {
     open.open(&path)
         .wrap_err(format!("couldn't write to file `{}`", path.display()))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
```

Comment:
Oh we should probably have a platform-independent test too, just for simple stuff. Ensures it works on macOS as well.

Reply:
does mac os have any quirks not found in *nix?